firefox-ios/Client/Frontend/Browser/WebView/PullRefreshView.swift

```diff
@@ -233,6 +233,13 @@ struct EasterEggViewLayoutBuilder {
     let sidePadding: CGFloat = 32.0
 
     func layoutEasterEggView(_ view: UIView, superview: UIView, isPortrait: Bool, isIpad: Bool) {
+        var isPortrait = isPortrait
+        // MARK: 667 is the screen height for iPhone SE 2/3 rd gen, 6,7,8.
+        // https://www.appmysite.com/blog/the-complete-guide-to-iphone-screen-resolutions-and-sizes/
+        if let screenHeight = UIWindow.keyWindow?.windowScene?.screen.bounds.height, screenHeight <= 667.0 {
```

Comment:
The comment is helpful but it might be worth moving this magic number into a constant to improve readability here.

Reply:
Yes sure, i'm moving it